app/views/templates/_template.html.slim

```diff
@@ -0,0 +1,11 @@
+tr
+  td class="question"
```

Comment:
@krisberry why did you put a tic if you have questions? You should put tick only if from your perspective everything is ok.